ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/SerializableSummary.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.ml.engine.utils;
```

Comment:
Seems this class only used by RCF summarizer. How about move it to `.../algorithms/clustering` package?

Reply:
Ok, will